docs/static-header-footer/bootstrap-3.html

```diff
@@ -42,6 +42,9 @@ <h2>Search</h2>
 						<label for="wb-srch-q" class="wb-inv">Search Canada.ca</label>
 						<input id="wb-srch-q" list="wb-srch-q-ac" class="wb-srch-q form-control" name="q" type="search" value="" size="34" maxlength="170" placeholder="Search Canada.ca">
 						<datalist id="wb-srch-q-ac">
+							data-wb-limit="10"
+							data-wb-filter-type="startWith"
```

Comment:
Why do we need does configuration?
```suggestion
```

Reply:
I copied the configuration from canada.ca.

<datalist id="datalist-sch-inp-ac" 
			data-wb-limit="10" 
			data-wb-filter-type="startWith" 
			data-wb-suggest="/content/dam/canada/srch-rchr/canada-en.json#/suggestions"> 
	</datalist>

Reply:
`<datalist id="datalist-sch-inp-ac" 
			data-wb-limit="10" 
			data-wb-filter-type="startWith" 
			data-wb-suggest="/content/dam/canada/srch-rchr/canada-en.json#/suggestions"> 
	</datalist>`

Reply:
Sorry, that is not a rational.

Reply:
What would be the right way to do it?

Reply:
Use it as per it's default configuration like:
```html
<datalist id="datalist-sch-inp-ac" data-wb-suggest="//www.canada.ca/content/dam/canada/srch-rchr/canada-en.json#/suggestions"> </datalist>
```

hum looking at the `id` of the `<datalist>` element, we might need to review it. It should be prefixed, like with "gc-" something. @GormFrank can you provide a suggestion for the `id` value?

Reply:
@ipaksc @duboisp I think the `id` should be something like: `gc-srch-q-list`

Reply:
I didn't saw we already had an id for the datalist, use this instead `wb-srch-q-ac`

Reply:
Removed